docs/source/markdown/podman-create.1.md

```diff
@@ -1412,6 +1412,20 @@ $ podman start --attach ctr
 b
 ```
 
+## CONMON
+
+When Podman starts a container it actually executes the conmon program, which
+then executes the OCI Runtime.  Conmon is the container monitor.  It is a small
+program whose job is to watch the primary process of the container, and if the
+container dies, save the exit code.  It also holds open the tty of the
+container, so that it can be attached to later. This is what allows Podman to
+run in detached mode (backgrounded), so Podman can exit but conmon continues to
+run.  Each container has their own instance of conmon. Conmon waits for the
+container to exit, gathers and saves the exit code, and then launches a Podman
+process to complete the container cleanup, by shutting down the network and
+storage.   For more information on common, please reference the common(8) man
```

Comment:
```suggestion
storage.   For more information on conmon, please reference the common(8) man
```

Reply:
I blame you for that, since I just cut and pased your previous message, at least I believe I did.